rides/templates/mail/opening.html

```diff
@@ -0,0 +1,15 @@
+{% extends "extend/email.html" %}
+
+{% block body %}
+    <div class="container main">
+        <h2>Hello {{ rider }},</h2>
+        <h3>There is an available ride for {{ event }}!</h3>
+        <p>Driver: {{ driver }}</p>
+        <p>If you wish to join this ride, please go to <a href={{ url }}>{{ url }}</a></p>
+        <p>Ensure that you were added to the ride, if not please hit the "Join Ride" button!</p>
+        {% if acc_type == "CSH" %}
+        <p>YOU CAN HAVE RIDEBOARD MESSAGE YOU ON SLACK INSTEAD OF EMAIL!</p>
```

Comment:
This probably doesn't need to be all caps, maybe include a linebreak before this, then
`Did you know RideBoard can message you on slack instead of emailing you? Link your slack account on EAC!`